include/bout/sys/expressionparser.hxx

```diff
@@ -129,7 +129,7 @@ private:
 
     LexInfo(const std::string& input, const std::string& reserved_chars = "");
 
-    /// Current token. -1 for number, -2 for string, 0 for "end of input"
+    /// Current token. -1 for number, -2 for symbol, -3 for {string}, 0 for "end of input"
```

Comment:
Idea for future PR: make this `variant<signed char, enum{Number, Symbol, String, EOF}>`?